### DISCUSSION TITLE
Why does the heat content of the reservoir change with the temperature of the injected water in a 1D linear thermal sweep model?

### DISCUSSION THREAD

**Discussion by huhanyu** [NatLabRockies/GEOPHIRES-X#161] [Q&A] [answered]
- In a 1D linear thermal sweep model, the reservoir properties are kept consistent, and only the temperature of the injected water is changed. However, the heat content of the reservoir varies greatly in the output data.
- Shouldn't the initial heat content of the reservoir only be related to the reservoir properties?
- Similar problems have been observed with multiple parallel fractures.
- If the output is correct, why does the temperature of the injected water change the heat content of the reservoir?

**The image shows an example of the change in reservoir thermal content after only modifying the injection water temperature in a 1-D linear thermal scanning model.**
![b2c7c2f3a5d9198bc004cc7147a6220](https://github.com/NREL/GEOPHIRES-X/assets/22694173/29fdbf47-2577-429b-a0e5-222516ddbda1)
![c3f6f1b4006f398f9544e43959264bb](https://github.com/NREL/GEOPHIRES-X/assets/22694173/045fea89-5314-4460-94db-a87b5dd3f5b2)

**Comment by softwareengineerprogrammer** [comment]
@kfbeckers or @malcolm-dsider can you field this one?

**Reply by kfbeckers** [reply, marked as answer]
@huhanyu, the reservoir heat content is calculated based on the initial reservoir temperature minus the injection temperature as the maximum amount of heat you can extract is cooling the rock down to the injection temperature. If you have a lower injection temperature, it means you can extract more heat and the reservoir heat content is higher.
![image](https://github.com/NREL/GEOPHIRES-X/assets/38221976/0f6122bd-de44-4edb-8ac1-20e472b8ddb4)

**Reply by huhanyu** [reply]
thanks，i got it！
